src/nativeMain/kotlin/commands/GetInfo.kt

```diff
@@ -0,0 +1,24 @@
+package commands
+
+import IEclairClient
+import IResultWriter
+import kotlinx.cli.ExperimentalCli
+import kotlinx.cli.*
+import kotlinx.coroutines.runBlocking
+
+@OptIn(ExperimentalCli::class)
+class GetInfoCommand(
+    private val resultWriter: IResultWriter,
+    private val eclairClient: IEclairClient
+): Subcommand("getinfo", "Get information about this instance such as version, features, nodeId and current block height as seen by the eclair.") {
+    private var password by option(ArgType.String, shortName = "p", description = "Password for the Eclair API").required()
+    private var host by option(ArgType.String, shortName = "l", description = "Host URL for the Eclair API").default("http://localhost:8080")
```

Comment:
- The shortname for the host is kept `l` because when I'm creating it as `h` I get the following:
```bash
claddy@claddy:~/Documents/eclair-cli$ eclair-cli.kexe getinfo -p password -h http://localhost:8080
Uncaught Kotlin exception: kotlin.IllegalStateException: Option with short name h was already added.
    at 0   eclair-cli.kexe                     0x557c34           kfun:kotlinx.cli.ArgParser#parse(kotlin.collections.List<kotlin.String>){}kotlinx.cli.ArgParserResult + 10196 
    at 1   eclair-cli.kexe                     0x554b5f           kfun:kotlinx.cli.ArgParser#saveOptionWithoutParameter(kotlinx.cli.ParsingValue<*,*>){} + 431 
    at 2   eclair-cli.kexe                     0x554c80           kfun:kotlinx.cli.ArgParser.saveStandardOptionForm#internal + 240 
    at 3   eclair-cli.kexe                     0x554e4c           kfun:kotlinx.cli.ArgParser.recognizeAndSaveOptionShortForm#internal + 348 
    at 4   eclair-cli.kexe                     0x556bc4           kfun:kotlinx.cli.ArgParser#parse(kotlin.collections.List<kotlin.String>){}kotlinx.cli.ArgParserResult + 5988 
    at 5   eclair-cli.kexe                     0x55c076           kfun:#main(kotlin.Array<kotlin.String>){} + 1750 
    at 6   eclair-cli.kexe                     0x588f47           Init_and_run_start + 503 
    at 7   eclair-cli.kexe                     0x58903a           Konan_main + 10 
    at 8   libc.so.6                           0x7f9ada029d8f     0x0 + 140303059295631 
    at 9   libc.so.6                           0x7f9ada029e3f     __libc_start_main + 127 
    at 10  eclair-cli.kexe                     0x415962           0x0 + 4282722 
Aborted (core dumped)
```

Reply:
I guess that's because `-h` is always reserved for the default `help` handler.

Do you really need to define a `shortName`? Can't we just use `--host` without any `shortName`? That would be better since `-l` doesn't make any sense for a `host` parameter.

Reply:
Okay, I'll remove the shortName.
I just tried if `l` could make sense as it might refer to `localhost`